### PR TITLE
feat:  support multi  files upload & update dependency

### PR DESCRIPTION
**Pull request by flywukong**
### Description
1) support uploading multiple files with one command， “object put  file1 ...  gnfd://bucket"
2) support uploading files in folders by one command with the "--recursive" flag. This command will create the same directory structure in the bucket.

### Rationale

improve cmd usage

### Example

image

**upload the folder and list by cmd**

image

image

```

./gnfd-cmd object put testfile1 testfile2 testfile3 gnfd://test333
Please enter a passphrase now:
upload testfile1 to gnfd://test333
upload testfile2 to gnfd://test333
upload testfile3 to gnfd://test333

```

### Changes

Notable changes:
* support   upload multiple files

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -180,6 +180,28 @@ gnfd-cmd object get gnfd://gnfd-bucket/gnfd-object file-path
 The filepath can be a specific file path, a directory path, or not set at all. 
 If not set, the command will download the content to a file with the same name as the object name in the current directory. If it is set as a directory, the command will download the object file into the directory.
 
+(3) create empty folder
+
+Please note that the object name corresponding to the folder needs to start with "/" as suffix
```
Comment (alexgao001) [inline README.md:185]: `start with` -> `end with`

File: cmd/cmd_object.go [hunk 2]
```diff
@@ -224,71 +232,153 @@ $ gnfd-cmd object mirror --destChainId 97 --bucketName yourBucketName --objectNa
...
-	c, cancelCreateBucket := context.WithCancel(globalContext)
-	defer cancelCreateBucket()
+	if !fileInfo.IsDir() {
+		return errors.New("failed to parse folder path with recursive flag")
+	}
+	fileInfos := make([]os.FileInfo, 0)
+	filePaths := make([]string, 0)
+	listFolderErr := filepath.Walk(folderName, func(path string, info os.FileInfo, err error) error {
+		if !info.IsDir() {
+			fileInfos = append(fileInfos, info)
```
Comment (alexgao001) [inline cmd/cmd_object.go:359]: seems fileInfo is not being used below

Comment (flywukong) [reply cmd/cmd_object.go:359]: used the info.size as  the objectSize . ` uploadFile(.., info.Size())`

File: README.md (hunk 1 above)
Comment (flywukong) [reply README.md]: fixed

**Review by alexgao001 — APPROVED** [review]

**Pull request merged**